src/cml.js

```diff
@@ -474,10 +475,13 @@ class CML {
     await exec(await driver.updateGitConfig({ userName, userEmail, remote }));
     if (unshallow) {
       if ((await exec('git rev-parse --is-shallow-repository')) === 'true') {
-        await exec('git fetch --unshallow');
+        return await exec('git fetch --unshallow');
```

Comment:
isn't this better?

```suggestion
        fetchDepth = 0;
```

Also do we need to add `git rev-parse --is-shallow-repository` below?

Reply:
I don't think so, but I haven't played with this. does `git rev-parse --is-shallow-repository` return `true` if the depth was already say 3? we wouldn't want to circumvent being able to expand this out directly.

Reply:
Needed to prevent errors; fixed in #1246